app/answerOperations.js

```diff
@@ -0,0 +1,58 @@
+var Question = require('../models/question');
+var Answer = require('../models/answer');
+var textUtils = require('../utils/textutils');
+var tokenUtils = require('../utils/tokenutils');
```

Comment:
Both are being used. We need textUtils to see if string is empty (not sure how null vs empty works in javascript). We need tokenUtils to get the user object from the given token in order to save the Answer object.